cli/command/system/info.go

```diff
@@ -135,9 +135,11 @@ func prettyPrintInfo(dockerCli *command.DockerCli, info types.Info) error {
 		fmt.Fprintf(dockerCli.Out(), "Default Runtime: %s\n", info.DefaultRuntime)
 	}
 
-	fmt.Fprintf(dockerCli.Out(), "Security Options:")
-	ioutils.FprintfIfNotEmpty(dockerCli.Out(), " %s", strings.Join(info.SecurityOptions, " "))
-	fmt.Fprintf(dockerCli.Out(), "\n")
+	if info.OSType != "windows" {
+		fmt.Fprintf(dockerCli.Out(), "Security Options:")
+		ioutils.FprintfIfNotEmpty(dockerCli.Out(), " %s", strings.Join(info.SecurityOptions, " "))
+		fmt.Fprintf(dockerCli.Out(), "\n")
```

Comment:
I'll leave @thaJeztah @vdemeester and @justincormack to argue among yourselves :innocent:. See https://github.com/docker/docker/pull/26387#issuecomment-245460479